src/main/java/com/exasol/sql/expression/function/exasol/ExasolFunction.java

```diff
@@ -0,0 +1,119 @@
+package com.exasol.sql.expression.function.exasol;
+
+import java.util.*;
+
+import com.exasol.sql.expression.*;
+import com.exasol.sql.expression.function.Function;
+import com.exasol.sql.expression.function.FunctionName;
+
+public class ExasolFunction extends AbstractValueExpression implements Function {
```

Comment:
JavaDoc missing.

Reply:
I know :)

Reply:
I'm not quite sure about the class name. I'll think it over again and then add the docs